tests/arg_aliases.rs

```diff
@@ -15,7 +15,8 @@ FLAGS:
     -V, --version    Print version information
 
 OPTIONS:
-    -o, --opt <opt>    [aliases: visible]";
+    -o, --opt <opt>    [aliases: visible]
+";
```

Comment:
I'm wondering if all of these tests need to care about whitspace, rather than having the full help-specific tests focus it.

My concern is with the cost of a change and ensuring that cost is not unnecessarily burdensome.  For example, an arg alias test is only about a fraction of the overall output.  If someone makes a change within a different part of clap, it shouldn't break arg aliases tests but only tests related to what was changed.

(Personally, I'd prefer we find a way to make these tests only care about the part of the help that is relevant.)

Reply:
I'm not sure how this change would affect tests-breaking-at-a-distance. My main change was in tests/utils.rs, to stop us from stripping the trailing whitespace from assertions. Given that that file is used by all these tests, having to update all of them is expected.

Going forward with other changes in a particular area, such a wide sweep should not be necessary